migrations/20220218091359_fixed_campaign_type_model/migration.sql

```diff
@@ -0,0 +1,74 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `status` on the `expenses` table. All the data in the column will be lost.
+  - You are about to drop the column `address` on the `people` table. All the data in the column will be lost.
```

Comment:
Why do we drop the column `address` on the `people` table?

Reply:
I'll check it.

Reply:
Done.